### PR TITLE
Don't filter proposed entities by canonical

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

**Comment by ahemphill** [comment]
Dependent on https://github.com/google/digitalbuildings/pull/549/files

File: tools/scoring/README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
```
Comment (charbull) [inline tools/scoring/README.md:3]: can you point to an `F-Score` definition from the internet?

File: tools/scoring/README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+1. Filter out irrelevant entities (e.g. those in the solution which have a noncanonical type)
+2. Create pairs of entities from the _proposed_ and _solution_ lists by matching `cloud_device_id` (for reporting devices) or raw field names (for virtual devices)
```
Comment (charbull) [inline tools/scoring/README.md:6]: can you please add a reference to the cloud device id: https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#config-format

File: tools/scoring/README.md [hunk 3]
```diff
@@ -21,24 +27,58 @@ Example (from the `digitalbuildings` directory): `python3 tools/scoring/scorer.p
...
+
+Quantifies whether the proposed file included the correct [points](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#defining-translations) in each entity. **This considers only canonically typed solution entities.**
+
+### Entity Type Identification
+
+Quantifies whether the proposed file assigned the correct [type](https://github.com/google/digitalbuildings/blob/master/ontology/docs/ontology_config.md#entitytypes) to each entity. **This considers only canonically typed solution entities.**
+
+### Raw Field Selection
+
+Quantifies whether the correct [raw fields](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#defining-translations) were selected in the proposed file. **This considers only canonically typed reporting solution solution entities.**
```
Comment (charbull) [inline tools/scoring/README.md]: nit: typo repeated

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/scoring/README.md (hunk 3 above)
Comment (ahemphill) [reply tools/scoring/README.md]: Fixed.

File: tools/scoring/README.md (hunk 2 above)
Comment (ahemphill) [reply tools/scoring/README.md:6]: (Will address in https://github.com/google/digitalbuildings/pull/549)

File: tools/scoring/README.md (hunk 1 above)
Comment (ahemphill) [reply tools/scoring/README.md:3]: (Will address in https://github.com/google/digitalbuildings/pull/549)

Comment (ahemphill) [reply tools/scoring/README.md:3]: https://github.com/google/digitalbuildings/pull/549/commits/b97384fb499100da6f85080782d2fcd1bf89d237

File: tools/scoring/README.md (hunk 2 above)
Comment (ahemphill) [reply tools/scoring/README.md:6]: https://github.com/google/digitalbuildings/pull/549/commits/b97384fb499100da6f85080782d2fcd1bf89d237

**Comment by ahemphill** [comment]
Merged via https://github.com/google/digitalbuildings/pull/561. Closing

**Pull request closed without merging**